polaris-react/src/index.ts

```diff
@@ -400,6 +400,9 @@ export {
 export {ScrollLockManagerContext as _SECRET_INTERNAL_SCROLL_LOCK_MANAGER_CONTEXT} from './utilities/scroll-lock-manager';
 export {WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT} from './utilities/within-content-context';
 export {useEventListener} from './utilities/use-event-listener';
+export {handleMouseUpByBlurring} from './utilities/focus';
+export {wrapWithComponent} from './utilities/components';
+export {isElementOfType} from './utilities/components';
```

Comment:
I'm happy to remove these but if we don't any issue exported them it would allow us to prevent duplicating them in the admin.